package.json

```diff
@@ -87,7 +86,10 @@
         "!**/*.css.ts",
         "!catalog",
         "!scripts/",
-        "!node_modules"
+        "!node_modules",
+        "!**/testing/**",
```

Comment:
I think this was added to the wrong section. It should be in the top level "files" on line 46, not the build wireit command

Reply:
Oops. Fix incoming